recipe/meta.yaml

```diff
@@ -7,38 +7,54 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sklearn-pandas-{{ version }}.tar.gz
-  sha256: bf908ea0e384e132da04355c7db67bd4f8efe145f0c9cd9f14726ce899d27542
+  # Using GH to get the tests.
+  url: https://github.com/scikit-learn-contrib/sklearn-pandas/archive/refs/tags/v2.2.0.tar.gz
+  sha256: 33b5ac5f18ebbde9798944357db0af1289986e502763fc0f8392c31d62af1961
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -v --no-deps --no-build-isolation
+  skip: true  # [py<38]
 
 requirements:
   host:
     - pip
-    - python >=3.7
+    - python
+    - setuptools
+    - wheel
   run:
-    - python >=3.7
-    - scikit-learn >=0.23.0
+    - python
+    # Some tests fail with scikit-learn>=1.2
+    - scikit-learn >=0.23.0,<1.2
     - scipy >=1.5.1
     - pandas >=1.1.4
     - numpy >=1.18.1
 
 test:
+  source_files:
+    - tests
   imports:
     - sklearn_pandas
   commands:
     - pip check
+    - pytest -v tests
   requires:
     - pip
+    - python
+    - pytest
 
 about:
   home: https://github.com/scikit-learn-contrib/sklearn-pandas
   summary: Pandas integration with sklearn
+  description: >
```

Comment:
I've only seen `|` used in our other recipes and that's what the parser targets. I'm hoping to avoid supporting multiple variants, if possible.

If you'd prefer/need to keep that, I already have a ticket in the parser work to add support for the other multi-line characters.

Reply:
`>` is handy because it'll replace newlines with spaces (basically rendering these multiple lines into a single line).

Reply:
See https://yaml-multiline.info/.

Reply:
_sigh_ A boy can dream of less work to do :P 
No big deal, I'll get to it eventually

Reply:
The good news is that the linter didn't crash :)